### PR TITLE
Adds ocarina note shuffle options to shuffle either row of songs separately

### PR DESCRIPTION
**Pull request by Maplesstar**
Assuming I understood/wrote this correctly, it will verify that random songs intended to replace warp songs will not contain/be contained in any of the top row songs. Did run like 20 seeds and there wasn't any overlap with top row songs, so I think it's functional??

### CODE REVIEW

**Comment by Maplesstar** [comment]
Verified I didn't mess any of the options up by adding top song only shuffling. All the spoiler logs and seeds I generated to test all 4 setting options came back correct. Only thing I'm not clear on is how difficulty is affected by shuffling only one row or the other. But I don't even get how "difficulty" is even defined for song note sequences so I can't really do better there.

**Comment by fenhl** [comment]
I opened Maplesstar/OoT-Randomizer#2 with an alternative implementation that reduces code duplication and adds a sneaky extra feature of allowing people to plando some songs while leaving the rest vanilla.

**Comment by fenhl** [comment]
The other presets need to be updated as well.

**Comment by Zannick** [comment]
I'd like a better name for the top row songs than just "top row". Utility songs? Non-warp songs?

**Comment by ETR-BTF** [comment]
All songs have utility. I don't know musical terms but maybe there's a word we can use to describe the pattern of the top row/non-warp songs?

**Comment by Maplesstar** [comment]
I originally called them child songs but was told to change it since storms is learned as adult. I do like frog songs though.

File: SettingsList.py [hunk 1]
```diff
@@ -3721,15 +3721,22 @@ def __init__(self, name, gui_text, min, max, default, step=1,
...
+            'all':  'All Songs',
+        },
         gui_tooltip    = '''\
-                         Will need to memorize a new set of songs.
-                         Can be silly, but difficult. Songs are
-                         generally sensible, and warp songs are
-                         typically more difficult.
-                         ''',
+            Will need to memorize a new set of songs.
+            Can be silly, but difficult. Songs are
```
Comment (fenhl) [inline SettingsList.py:3736]:
```suggestion
            Can be silly, but difficult. Frog songs are
```

Comment (Maplesstar) [reply SettingsList.py:3736]: That's not what it means, it means all songs are generally sensible, just that warp songs are generally harder but still sensible.

Comment (fenhl) [reply SettingsList.py:3736]: Maybe “warp songs are typically more difficult than frog songs”?

Comment (Maplesstar) [reply SettingsList.py:3736]: I dunno, I didn't change the tooltip at all, that's just what it's always been.

Comment (fenhl) [reply SettingsList.py:3736]: Sure, but this would be a good opportunity to make it clearer.

Comment (Maplesstar) [reply SettingsList.py]: Yeah, alright, added that change

**Review by cjohnson57 — APPROVED** [review]
Helped a bit with this PR and all seems good to me, downloaded and tested and everything's working as expected.

**Pull request merged**